src/helpers/files.js

```diff
@@ -139,14 +139,46 @@ const registerFileActionFallback = () => {
 
 }
 
+const newRichWorkspaceFileMenuPlugin = {
+	attach(menu) {
+		const fileList = menu.fileList
+		const descriptionFile = t('text', 'Readme') + '.' + loadState('text', 'default_file_extension')
+		// only attach to main file list, public view is not supported yet
+		if (fileList.id !== 'files' && fileList.id !== 'files.public') {
+			return
+		}
+
+		// register the new menu entry
+		menu.addMenuEntry({
+			id: 'rich-workspace-init',
+			displayName: t('text', 'Add description'),
+			templateName: descriptionFile,
+			iconClass: 'icon-rename',
+			fileType: 'file',
+			useInput: descriptionFile,
```

Comment:
```suggestion
			useInput: false,
```

To make sure that the input is not shown